mne/viz/epochs.py

```diff
@@ -162,6 +162,17 @@ def plot_image_epochs(epochs, picks=None, sigma=0., vmin=None,
     return figs
 
 
+@deprecated
```

Comment:
Did you test this? I don't think it will work. Please add a test with `warnings.catch_warnings(record=True) as w:` making a call to this version to ensure it continues to at least run. I think you need to call it as a function and provide a reasonable string arguments (unless it is doing something magical under the hood), do `git grep "@dep"` and you'll see other instances of it.